### DISCUSSION TITLE
Menu icon not appearing

### DISCUSSION THREAD

**Discussion by ghost** [okwasniewski/MiniSim#75] [Q&A] [answered]
After install with brew the menu icon doesn't appear. I did a system restart.
OSX (12.7), Xcode (13.3.1), Android Studio (2022.2). What can I do now?

**Comment by okwasniewski** [comment]
Hey @hanskuiters, 

Can you confirm that MiniSim is running by opening the activity monitor and checking if it's there? And if it is then force quit it and restart.

**Reply by ghost** [reply, marked as answer]
Hi @okwasniewski , it wasn't running. Then I found that the install didn't run the app but placed a real app in de application directory. Running that app did the trick. :-)
